src/js/utils/settings-api.js

```diff
@@ -0,0 +1,59 @@
+import Promise from 'bluebird';
+import localforage from 'localforage';
+import assign from 'object-assign';
+import cloneDeep from 'lodash.clonedeep';
+
+import {
+  INITIAL_STORE,
+  SETTINGS_BACKEND_STORE_NAME,
+  SETTINGS_NOTE_CONFIG } from '../constants/constants';
+
+const settingsStore = localforage.createInstance({
+  name: SETTINGS_BACKEND_STORE_NAME
+});
+const initialStoreSettings = cloneDeep(INITIAL_STORE)[SETTINGS_BACKEND_STORE_NAME];
```

Comment:
Line is too long.<br>'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).